stac_fastapi/types/stac_fastapi/types/aggregation.py

```diff
@@ -0,0 +1,45 @@
+"""Aggregation Extension types."""
```

Comment:
TBH I don't think we want `extension` in stac_fastapi.types, if we start adding one we would be questioned to add more, which I personally can't promise to maintain. I would prefer if we keep every `extension` code in `stac_fastapi.extensions`

Reply:
I moved this to `stac_fastapi.extensions.core.aggregation.types`